charts/camunda-platform-alpha/templates/core/configmap.yaml

```diff
@@ -263,7 +271,9 @@ data:
           # Transport port
           port: {{ .Values.global.elasticsearch.url.port }}
           # Index prefix, configured in Zeebe Elasticsearch exporter
-          prefix: {{ .Values.global.elasticsearch.prefix }}
+          {{- if .Values.core.index.prefix }}
+          prefix: {{ .Values.core.index.prefix | quote }}
+          {{- end }}
```

Comment:
❌ I think here we wanted to keep the other prefix right?

Reply:
Fixed in: https://github.com/camunda/camunda-platform-helm/commit/ae9c82512781c1c307eb20a96589d0b7575aa3e0